### PR TITLE
Code: reduce progress notification spam

### PR DESCRIPTION
**Pull request by lnicola**

### CODE REVIEW

**Comment by lnicola** [comment]
Untested, let's not merge this today.

File: editors/code/src/net.ts [hunk 1]
```diff
@@ -99,13 +99,15 @@ export async function download(opts: DownloadOpts) {
...
-                progress.report({
-                    message: newPercentage.toFixed(0) + "%",
-                    increment: newPercentage - lastPercentage
-                });
-
-                lastPercentage = newPercentage;
+                const newPercentage = Math.round((readBytes / totalBytes) * 100);
+                if (newPercentage !== lastPercentage) {
+                    progress.report({
+                        message: newPercentage.toFixed(0) + "%",
```
Comment (IWANABETHATGUY) [inline editors/code/src/net.ts:105]: use template literal here may be better ?
```js
`${newPercentage.toFixed(0)}%`
```

Comment (lnicola) [reply editors/code/src/net.ts]: Eh, sure, why not.

**Comment by lnicola** [comment]
There's too much variation in the download speeds for me to see a difference here, but it can't hurt.

bors r+

**Pull request merged**